### PR TITLE
Update Windows docker host with AMI and M54xlarge from #417

### PR DESCRIPTION
**Pull request by peterzhuamazon**
### Description
Update Windows docker host with AMI and M54xlarge from #417

### Issues Resolved
https://github.com/opensearch-project/opensearch-ci/issues/412

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: lib/compute/agent-nodes.ts [hunk 1]
```diff
@@ -217,13 +217,13 @@ export class AgentNodes {
     this.WINDOWS2019_X64_DOCKER_HOST = {
       agentType: 'windows',
       customDeviceMapping: '/dev/sda1=:600:true:::encrypted',
-      workerLabelString: 'Jenkins-Agent-Windows2019-X64-C54xlarge-Docker-Host',
-      instanceType: 'C54xlarge',
+      workerLabelString: 'Jenkins-Agent-Windows2019-X64-M54xlarge-Docker-Host',
```
[the remark this answers is not in the document]
Comment (peterzhuamazon) [reply lib/compute/agent-nodes.ts:220]: Will send the build repo PR once this is deployed.

**Review by gaiksaya — APPROVED** [review]
Please confirm that the AMI is publicly accessible?

**Comment by peterzhuamazon** [comment]
> Please confirm that the AMI is publicly accessible?

Yes, it is.

**Comment by gaiksaya** [comment]
I think we can add tests for confirming whether the AMI is available or not by trying to get it in ci tests

**Pull request merged**